go/felix/set/set.go

```diff
@@ -56,6 +70,14 @@ func (set mapSet) Add(item interface{}) {
 	set[item] = emptyValue
 }
 
+func (set mapSet) AddAll(itemArray interface{}) {
+
+	arrVal := reflect.ValueOf(itemArray)
```

Comment:
Yes, Go's arrays aren't covariant (think that's the word) so you can't pass a `[]foo` to a function that wants a `[]interface{}`.  Only way to handle that is with `reflect`, which I think is OK in such a datastructure.